m2c/main.py

```diff
@@ -662,6 +674,23 @@ def parse_flags(flags: List[str]) -> Options:
         struct_name, field_name = parts
         union_field_overrides[struct_name] = field_name
 
+    # Parse void variable type overrides
+    void_var_type_overrides: Dict[str, str] = {}
+    for override in args.void_var_types:
+        parts = override.split(":", 1)
+        if len(parts) != 2:
+            print(
+                f"Error: Invalid void variable type override '{override}'. "
+                "Expected format: VAR_NAME:TYPE_NAME",
+                file=sys.stderr,
+            )
+            sys.exit(1)
+        var_name, type_name = parts
+        # Add implicit * if not already present (unless it's already a pointer type)
+        if not type_name.endswith("*"):
+            type_name = type_name + "*"
```

Comment:
this is a bit controversial; the justification is that, unless you wrap it in quotes, `*` will be interpreted as a glob by your shell. I don't love this solution, though, so I'm open to better suggestions.